index.js

```diff
@@ -114,10 +177,24 @@ class Bot extends EventEmitter {
           this._handleEvent('delivery', event)
         }
 
+        // handle message read
+        if (event.read) {
+          this._handleEvent('read', event)
+        }
+
         // handle authentication
         if (event.optin) {
           this._handleEvent('authentication', event)
         }
+
+        // handle account_linking
+        if (event.account_linking && event.account_linking.status) {
+          if (event.account_linking.status == 'linked') {
+            this._handleEvent('accountLinked', event)
+          } else if (event.account_linking.status == 'unlinked') {
+            this._handleEvent('accountUnlinked', event)
+          }
+        }
```

Comment:
@charlesaraya you are welcome to do a pull request containing the specific changes to eXeDK:messenger-bot/master